site/layouts/shortcodes/include.html

```diff
@@ -1,6 +1,6 @@
 {{ $file := .Get 0 }}
-[{{$file}}]({{printf "/examples/%s" $file}})
-{{ $path := (printf "../../static/examples/%s" $file)}}
+&#128462; [{{$file}}]({{printf "/examples/%s" $file}})
```

Comment:
I can not see the icon even if using Chrome with private mode.
It's curious.
<img width="799" alt="screenshot2023-08-17 4 46 16" src="https://github.com/kubernetes-sigs/kueue/assets/68272500/bfdcb7a6-6f39-48ce-bb90-70754176b4ee">

Reply:
It might be that the font in your machine doesn't have the unicode character :(

Let's remove it for now and have a separate issue to have an image icon. And maybe we can do some additional formatting.

Reply:
> It might be that the font in your machine doesn't have the unicode character :(

I tried to open the page on the IPad and MacBook. However, I can not see the icon in both devices :(

Reply:
https://github.com/kubernetes-sigs/kueue/issues/1065 Opened this to follow up